recipes/sqlite3/all/conanfile.py

```diff
@@ -22,19 +22,21 @@ class ConanSqlite3(ConanFile):
                "enable_fts5": [True, False],
                "enable_json1": [True, False],
                "enable_rtree": [True, False],
-               "omit_load_extension": [True, False]
+               "omit_load_extension": [True, False],
+               "enable_unlock_notify": [True, False]
                }
     default_options = {"shared": False,
                        "fPIC": True,
                        "threadsafe": 1,
-                       "enable_column_metadata": False,
+                       "enable_column_metadata": True,
```

Comment:
usually we enable only the default option with same behavior in the upstream. Is there any reason to change it ?

Reply:
yes : https://github.com/conan-io/conan-center-index/issues/413

Reply:
fair enough!